dev-client/src/screens/AddUserToProjectScreen/components/AddTeamMemberForm.tsx

```diff
@@ -64,7 +67,10 @@ export const AddTeamMemberForm = ({projectId}: FormProps) => {
       // Success
       else {
         const user = userOrError as UserFields;
-        navigation.navigate('ADD_USER_PROJECT_ROLE', {projectId, user});
+        const userId = user.id;
+        // Currently there is no need to access any user's preferences besides the current user
```

Comment:
I'm thinking of making a tech debt item to address this weirdness -- I don't think we should need to know any user preferences except for the current user, so perhaps the set of users should not include user preferences, and the current user's preferences should be stored in a different fashion?